playwright/tests/reader-settings.spec.ts

```diff
@@ -0,0 +1,269 @@
+import { test, expect } from '@playwright/test';
+import { WebReaderPage } from '../pageobjects/web-reader.page.ts';
+
+test.describe('Test HTML pub', () => {
+  test.beforeEach(async ({ page }) => {
+    // goto
+  });
+
+  test('Confirm reader settings are visible', async ({ page }) => {
+    const webReaderPage = new WebReaderPage(page);
+    await webReaderPage.loadPage(
+      'https://nypl-web-reader.vercel.app/html/moby-epub3'
+    ); // temporary
+    await expect(webReaderPage.fullScreenButton).toBeVisible();
+    await webReaderPage.settingsButton.click();
+    await expect(webReaderPage.defaultFont).toBeVisible();
+    await expect(webReaderPage.serifFont).toBeVisible();
+    await expect(webReaderPage.sansSerifFont).toBeVisible();
+    await expect(webReaderPage.dyslexiaFont).toBeVisible();
+    await expect(webReaderPage.whiteBackground).toBeVisible();
+    await expect(webReaderPage.sepiaBackground).toBeVisible();
+    await expect(webReaderPage.blackBackground).toBeVisible();
+    await expect(webReaderPage.resetTextSize).toBeVisible();
+    await expect(webReaderPage.decreaseTextSize).toBeVisible();
+    await expect(webReaderPage.increaseTextSize).toBeVisible();
+    await expect(webReaderPage.paginatedStyle).toBeVisible();
+    await expect(webReaderPage.scrollingStyle).toBeVisible();
+    await expect(webReaderPage.zoomInButton).not.toBeVisible();
+  });
+
+  test('Open and close reader settings', async ({ page }) => {
+    const webReaderPage = new WebReaderPage(page);
+    await webReaderPage.loadPage(
+      'https://nypl-web-reader.vercel.app/html/moby-epub3'
+    ); // temporary
+    await webReaderPage.settingsButton.click();
+    await expect(webReaderPage.defaultFont).toBeVisible();
+    await webReaderPage.settingsButton.click();
+    await expect(webReaderPage.defaultFont).not.toBeVisible();
+  });
+
+  test('Displays default settings', async ({ page }) => {
+    const webReaderPage = new WebReaderPage(page);
+    await webReaderPage.loadPage(
+      'https://nypl-web-reader.vercel.app/html/moby-epub3'
+    ); // temporary
+    await webReaderPage.settingsButton.click();
+    await expect(webReaderPage.defaultFont).toBeChecked();
+    await expect(webReaderPage.whiteBackground).toBeChecked();
+    //await expect(webReaderPage.getTextSize).toBe('100%'); // expect --USER__fontSize: 100%
```

Comment:
The `getTextSize` is an async function, so this would need to be awaited and invoked.

```suggestion
    await expect(await webReaderPage.getTextSize()).toBe('96%'); 
```

Reply:
Omg this works now!! Thanks so much!